src/main/java/org/apache/commons/io/input/BoundedReader.java

```diff
@@ -121,6 +123,7 @@ public int read() throws IOException {
      */
     @Override
     public int read(final char[] cbuf, final int off, final int len) throws IOException {
+        IOUtils.checkFromIndexSize(cbuf, off, len);
```

Comment:
Javadoc `@throws` IndexOutOfBoundsException is missing.

Reply:
Fixed in https://github.com/apache/commons-io/pull/790/commits/8d520f72c3698440e2275b860a991a41757ad2fb